extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/InfinispanClientProcessor.java

```diff
@@ -71,6 +74,11 @@ class InfinispanClientProcessor {
      */
     InfinispanClientBuildTimeConfig infinispanClient;
 
+    @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
```

Comment:
just curious, but isn't this "onlyIf" condition redundant since I'd assume none would otherwise consume a `NativeImageFeatureBuildItem` ?

Reply:
I just copied the pattern used elsewhere.